src/pages/views/explorer/explorerChart.tsx

```diff
@@ -253,7 +257,7 @@ class ExplorerChartBase extends React.Component<ExplorerChartProps> {
       <>
         <div style={styles.titleContainer}>
           <Title headingLevel="h3" size="md">
-            {t(this.getChartTitle(perspective))}
+            {intl.formatMessage(messages.PerspectiveValues, { value: perspective })}
```

Comment:
The chart titles are not the same as the perspective. They should contain the words "Top 5 Costliest"

```
"aws": "Amazon Web Services - Top 5 Costliest",
"aws_ocp": "Amazon Web Services filtered by OpenShift - Top 5 Costliest",
"azure": "Microsoft Azure - Top 5 Costliest",
"azure_ocp": "Microsoft Azure filtered by OpenShift - Top 5 Costliest",
"gcp": "Google Cloud Platform - Top 5 Costliest",
"gcp_ocp": "Google Cloud Platform filtered by OpenShift - Top 5 Costliest",
"ibm": "IBM Cloud - Top 5 Costliest",
"ocp": "All OpenShift cost - Top 5 Costliest",
"ocp_cloud": "All cloud filtered by OpenShift - Top 5 Costliest"
```

Reply:
Done